processing/src/main/java/org/apache/druid/query/scan/ScanQueryEngine.java

```diff
@@ -266,7 +261,7 @@ public void cleanup(Iterator<ScanResultValue> iterFromMake)
   private long calculateLimit(ScanQuery query, ResponseContext responseContext)
```

Comment:
I think it would be clearer to call this method "calculateRemainingScanRowsLimit"

Reply:
Renamed